examples/experimental/v1alpha2/backendtlspolicy-system-certs.yaml

```
@@ -0,0 +1,14 @@
+#$ Used in:
+#$ - site-src/guides/tls.md
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: BackendTLSPolicy
+metadata:
+  name: tls-upstream-dev
+spec:
+  targetRef:
+    kind: Service
+    name: dev-service
+    group: ""
+  tls:
+    wellKnownCACerts: "System"
+    hostname: dev-pod.example.com
```

Same comment as above
```suggestion
    hostname: dev.example.com
```